app/src/main/java/com/github/geohunt/app/maps/marker/MarkerCreator.kt

```diff
@@ -0,0 +1,91 @@
+package com.github.geohunt.app.maps.marker
+
+import android.graphics.Bitmap
+import android.graphics.Color
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import com.github.geohunt.app.R
+import com.google.android.gms.maps.model.BitmapDescriptorFactory
+import com.google.android.gms.maps.model.LatLng
+import com.google.maps.android.compose.MarkerInfoWindowContent
+import com.google.maps.android.compose.rememberMarkerState
+import java.time.LocalDateTime
+import java.time.Month
+
+//Hardcoded list used to test correct display of events on the map
+private val mockBitmap: Bitmap = Bitmap.createBitmap(IntArray(120*120){ Color.CYAN}, 90, 90, Bitmap.Config.ARGB_8888)
+private val mockChallengeDatabase : List<MarkerData> = listOf(
+    MarkerData("Event 1", mockBitmap, LatLng(46.51958, 6.56398), LocalDateTime.of(2023, Month.MAY, 1, 19, 39, 12)),
+    MarkerData("Event 2", mockBitmap, LatLng(46.52064, 6.56780), LocalDateTime.of(2023, Month.MAY, 2, 12, 24, 35)),
+    MarkerData("Event 3", mockBitmap, LatLng(46.51881, 6.56779), LocalDateTime.of(2023, Month.MAY, 3, 16, 12, 12))
+)
+
+/**
+ * Adds the data from the database to the map as markers
+ */
+@Composable
+fun DisplayMarkers() {
+    mockChallengeDatabase.forEach { challenge ->
+        MarkerInfoWindowContent(
+            state = rememberMarkerState(position = challenge.coordinates),
+            title = challenge.title,
+            snippet = challenge.expiryDate.toString(),
+            icon = BitmapDescriptorFactory.defaultMarker(BitmapDescriptorFactory.HUE_AZURE)
```

Comment:
The color should I think follow the colors that we defined. Try something like orange.